rke/k8s_rke_system_images.go

```diff
@@ -6263,9 +6263,9 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
 		// Enabled out of band post v2.6.3
-		"v1.20.14-rancher2-1": {
+		"v1.20.15-rancher1-1": {
```

Comment:
@kinarashah Does this mean that the kubernetes versions in `templates.go` will have to be bumped as well? Coredns restricts for 1.20.14 and 1.21.8

Reply:
@annablender You're right, tho in this case the template conditions are `>=1.20.14` so it'll cover new versions moving forward.